config/packages/package-operator/components/remote-phase/.test-fixtures/affinity-tolerations-resources/package-operator-remote-phase-manager.Deployment.yaml

```diff
@@ -20,8 +20,8 @@ spec:
         app.kubernetes.io/name: package-operator-remote-phase-manager
         hypershift.openshift.io/need-management-kas-access: "true"
     spec:
-      affinity: {"nodeAffinity":{"requiredDuringSchedulingIgnoredDuringExecution":{"nodeSelectorTerms":[{"matchExpressions":[{"key":"node-role.kubernetes.io/infra","operator":"Exists"}]}]}}}
-      tolerations: [{"effect":"NoSchedule","key":"node-role.kubernetes.io/infra"}]
```

Comment:
Do you think we could maybe remove this hardcoding here, replace it with templating from `.spec.config` and then configure all hypershift labels downstream?